SoundSwitch/Localization/SettingsStrings.resx

```diff
@@ -232,7 +232,8 @@
     <value>Windows Notification: The default windows tray pop-up.
 Sound Notification: Plays a sound on the switched Playback Device.
 Customized Sound Notification: Plays the specified sound on the switched Playback Device.
-Toast Notification: Uses a special form of the windows tray pop-up version.</value>
+Toast Notification: Uses a special form of the windows tray pop-up version.
+Banner: Uses a custom always-on-top form, may be useful for in-game usage because it stands above directdraw/opengl.</value>
```

Comment:
" because it stands above directdraw/opengl" can be removed, the average gamer doesn't need this information.
`Banner: Uses a custom always-on-top frame, useful for in-game usage.`